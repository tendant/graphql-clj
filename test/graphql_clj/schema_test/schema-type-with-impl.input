interface Pet {
  name: String
}
type Dog implements Pet {
  name: String!
}
