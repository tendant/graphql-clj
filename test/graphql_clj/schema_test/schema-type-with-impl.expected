{:interfaces {:Pet {:fields {:name {:type String}}}},
 :objects
 {:Dog {:implements [:Pet], :fields {:name {:type (non-null String)}}}}}
